roles/openshift_health_checker/openshift_checks/etcd_traffic.py

```diff
@@ -0,0 +1,59 @@
+"""Module that scans journalctl for messages caused as a symptom of increased etcd traffic"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdTraffic(OpenShiftCheck):
+    """Check if host is being affected by an increase in etcd traffic."""
+
+    name = "etcd_traffic"
+    tags = ["health", "etcd"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        """Skip hosts that do not have etcd or masters in their group names."""
+        master_count = get_var(task_vars, "openshift", "master", "master_count", default=1)
+        group_names = get_var(task_vars, "group_names", default=[])
+        active = "masters" in group_names
+        if int(master_count) > 1:
+            active = "etcd" in group_names
+
+        return super(EtcdTraffic, cls).is_active(task_vars) and active
+
+    def run(self, tmp, task_vars):
+        master_count = get_var(task_vars, "openshift", "master", "master_count", default=1)
+        deploy_type = get_var(task_vars, "openshift", "common", "service_type")
+        unit = self._get_systemd_unit(master_count, deploy_type)
+
+        log_matchers = [{
+            "start_regexp": r"Starting Etcd Server",
```

Comment:
I loosely based most of the logic in this check on Luke's [implementation in oadm diagnostics](https://github.com/openshift/origin/blob/master/pkg/diagnostics/systemd/systemd.go#L64), which also uses a field to mark the beginning of the log set that we want to parse through. This is just to limit the search to last unit startup.